kernel_hardening_checker/checks.py

```diff
@@ -380,8 +380,9 @@ def add_kconfig_checks(l, arch):
     l += [KconfigCheck('cut_attack_surface', 'clipos', 'MAGIC_SYSRQ', 'is not set')]
     l += [KconfigCheck('cut_attack_surface', 'clipos', 'KEXEC_FILE', 'is not set')] # refers to LOCKDOWN (permissive)
     l += [KconfigCheck('cut_attack_surface', 'clipos', 'USER_NS', 'is not set')] # user.max_user_namespaces=0
-    l += [KconfigCheck('cut_attack_surface', 'clipos', 'X86_CPUID', 'is not set')]
-    l += [KconfigCheck('cut_attack_surface', 'clipos', 'X86_IOPL_IOPERM', 'is not set')] # refers to LOCKDOWN
+    if arch in ('X86_64', 'X86_32'):
```

Comment:
Hi @jvoisin,

For the `cut_attack_surface` checks we don't add `if arch` to simplify maintainance.

The desired values are `is not set` anyway.

Reply:
It would be nice to have a comment in the code to make this explicit then :)

Reply:
Done: https://github.com/a13xp0p0v/kernel-hardening-checker/commit/40b32a181e9657b2af3bd9ecb97c71c5fe855204